pages/examples/sensible-aria-usage/current/_examples/marking-an-element-as-current-using-aria-current/index.html

```diff
@@ -1,14 +1,14 @@
 <ul>
   <li>
-    <a href="...">Home</a>
+    <a href="">Home</a>
```

Comment:
Same here (and some more below).

Reply:
I decided to undo all of them (back to "..."). I'm happy to take another look at this if you don't agree.